bootstrap.egg-info/entry_points.txt

```diff
@@ -2,6 +2,11 @@
 egg_info = setuptools.command.egg_info:egg_info
 build_py = setuptools.command.build_py:build_py
 sdist = setuptools.command.sdist:sdist
+dist_info = setuptools.command.dist_info:dist_info
+develop = setuptools.command.develop:develop
+install_scripts = setuptools.command.install_scripts:install_scripts
+bdist_egg = setuptools.command.bdist_egg:bdist_egg
+install_egg_info = setuptools.command.install_egg_info:install_egg_info
```

Comment:
Do you know why these new entries are needed? My goal with the bootstrap was to keep it as lean as possible, to provide just enough functionality that setuptools can run `egg_info` on itself to generate the full metadata.

Reply:
I cannot say with 100% certainty why they were needed, I just noticed the errors when I tried obtaining the dist object using `distutils.core.run_setup` and then fixed them one by one by trial and error 😢 

But since it was brought to my attention that `pybind11` would fail with that approach, I decided to avoid any changes in the `build_meta` and rely only on the internals of the distribution object and therefore revert these changes.

🤞 With a bit of luck the tests will pass!